parlai/agents/drqa/utils.py

```diff
@@ -70,6 +88,8 @@ def vectorize(opt, ex, word_dict, feature_dict):
     # Create extra features vector
     features = torch.zeros(len(ex['document']), len(feature_dict))
 
+    spacy_doc = NLP(' '.join(ex['document']))
```

Comment:
this should not be done unless the ner and pos tags are actually used (its expensive)